### PR TITLE
BigQuery connectors

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Setups connectors to update BigQuery devices, sites and airqlouds.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [OPS-113](https://airqoteam.atlassian.net/browse/OPS-113)

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

[OPS-113]: https://airqoteam.atlassian.net/browse/OPS-113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/kafka-connectors/bigquery-connector/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+COPY . $PROJECT_DIR/
+
+FROM base as sites-source-connector
+CMD ["python", "main.py", "--target=sites-source-connector"]
+
+FROM base as devices-source-connector
+CMD ["python", "main.py", "--target=devices-source-connector"]
+
+FROM base as airqlouds-source-connector
+CMD ["python", "main.py", "--target=airqlouds-source-connector"]
```
Comment (NoahNsimbe) [inline src/kafka-connectors/bigquery-connector/Dockerfile:21]: @123MwanjeMike all these need to run on separate containers.

File: src/kafka-connectors/bigquery-connector/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+DEVICES_TABLE=
+SITES_TABLE=
+AIRQLOUDS_SITES_TABLE=
+AIRQLOUDS_TABLE=
+BOOTSTRAP_SERVERS=
+AIRQLOUDS_TOPIC=
+SITES_TOPIC=
+DEVICES_TOPIC=
+GOOGL_APPLICATION_CREDENTIALS=google_application_credentials.json
+```
```
Comment (NoahNsimbe) [inline src/kafka-connectors/bigquery-connector/README.md:21]: @123MwanjeMike sample of contents of the .env

**Review by Mnoble-19 — APPROVED** [review]

File: src/kafka-connectors/bigquery-connector/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| `GOOGLE_APPLICATION_CREDENTIALS` | **Required**. Points to a google credentials file with permissions to update BigQuery tables |
+| `DEVICES_TABLE`                  | **Required**. BigQuery table with devices                                                    |
+| `SITES_TABLE`                    | **Required**. BigQuery table with sites                                                      |
+| `AIRQLOUDS_SITES_TABLE`          | **Required**. BigQuery table with sites and airqlouds                                        |
+| `AIRQLOUDS_TABLE`                | **Required**. BigQuery table with airqlouds                                                  |
+| `BOOTSTRAP_SERVERS`              | **Required**. Bootstrap servers                                                              |
+| `AIRQLOUDS_TOPIC`                | **Required**. AirQlouds topic                                                                |
+| `SITES_TOPIC`                    | **Required**. Sites topic                                                                    |
+| `DEVICES_TOPIC`                  | **Required**. Devices topic                                                                  |
```
Comment (NoahNsimbe) [inline src/kafka-connectors/bigquery-connector/README.md:22]: @123MwanjeMike sample of contents of the .env

Comment (123MwanjeMike) [reply src/kafka-connectors/bigquery-connector/README.md:22]: thanks @NoahNsimbe 
Adding it as a .env.sample

**Review by 123MwanjeMike — APPROVED** [review]

**Pull request merged**